centreon/src/App/ResourceConfiguration/Infrastructure/ApiPlatform/Resource/GlobalMacroResource.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/*
+ * Copyright 2005 - 2025 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+declare(strict_types=1);
+
+namespace App\ResourceConfiguration\Infrastructure\ApiPlatform\Resource;
+
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\GetCollection;
+use ApiPlatform\OpenApi\Model;
+use App\ResourceConfiguration\Domain\Security\GlobalMacroPermissionEnum;
+use App\ResourceConfiguration\Infrastructure\ApiPlatform\State\ListGlobalMacrosProvider;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[ApiResource(
+    shortName: 'GlobalMacro',
+    operations: [
+        new GetCollection(
+            uriTemplate: '/configuration/macros/globals',
```

Comment:
A naming policy could worth it. 
`/` separated parts should have a clear meaning.

Example:

```
/articles                       GET articles collection
/articles/1                     GET article item
/articles/1/comments            GET comments collection of article 1
/articles/1/comments/123        GET comment item of article 1
/articles/1/comments/123/likes  GET likes collection of comment123 of article 1
...
```